include/itkWASMMeshToMeshFilter.hxx

```diff
@@ -205,10 +205,10 @@ WASMMeshToMeshFilter<TMesh>
     throw std::runtime_error("Unexpected cell pixel type");
   }
 
-  using CoordRepType = typename MeshType::CoordRepType;
+  using PointType = typename MeshType::PointType;
   const rapidjson::Value & pointsJson = document["points"];
   const std::string pointsString( pointsJson.GetString() );
-  CoordRepType * pointsPtr = reinterpret_cast< CoordRepType * >( std::atol(pointsString.substr(35).c_str()) );
+  const auto * pointsPtr = reinterpret_cast< PointType * >( std::atol(pointsString.substr(35).c_str()) );
```

Comment:
I still wonder if it's OK to represent a pointer by a signed long integer, as retrieved by `std::atol`. On Windows, a `long`is only 32 bits, you know. Maybe better use `std::strtoull` instead of  `std::atol`?  `std::strtoull` converts to 64-bit unsigned long long, which should be sufficient for a `PointType *` pointer on any platform. Anyway, just my two cents!

Reply:
Good thoughts, @N-Dekker . WASM is current 32-bit, but this could be improved for the future. I created #515 to track.